impl/asm-based-transformer/src/main/java/org/wildfly/transformer/asm/TransformerImpl.java

```diff
@@ -226,6 +228,86 @@ public void visitMethodInsn(int opcode, String owner, String name, String desc,
                         mv.visitMethodInsn(opcode, owner, name, desc, itf);
                     }
 
+                    @Override
+                    public void visitInvokeDynamicInsn(String name, String desc, Handle bootstrapMethodHandle, Object... bootstrapMethodArguments) {
+                        final String descOrig = desc;
+                        desc = replaceJavaXwithJakarta(desc);
+                        final String ownerOrig = bootstrapMethodHandle.getOwner();
+                        String bootstrapMethodHandleOwner = replaceJavaXwithJakarta(ownerOrig);
+                        final String bootstrapMethodHandleDescOrig = bootstrapMethodHandle.getDesc(); 
+                        String bootstrapMethodHandleDesc = replaceJavaXwithJakarta(bootstrapMethodHandleDescOrig);
+                        if (!descOrig.equals(desc)) {  // if we are changing
+                            // mark the class as transformed
+                            setClassTransformed(true);
+                        }
+                        if (!ownerOrig.equals(bootstrapMethodHandleOwner) ||
+                        !bootstrapMethodHandleDescOrig.equals(bootstrapMethodHandleDesc)) {  // if we are changing
+                            // mark the class as transformed
+                            setClassTransformed(true);
+                            bootstrapMethodHandle = new Handle(
+                                    bootstrapMethodHandle.getTag(),
+                                    bootstrapMethodHandleOwner,
+                                    bootstrapMethodHandle.getName(),                                     
+                                    bootstrapMethodHandleDesc,
+                                    bootstrapMethodHandle.isInterface());
+                        }
+                        /** bootstrapMethodArguments can be Integer, Float,Long, Double,String, Type, Handle, ConstantDynamic value. 
+                         * ConstantDynamic can modify content of array at runtime.
+                         */
+                        Object[] copyBootstrapMethodArguments = null;
+                        for(int looper = 0; looper < bootstrapMethodArguments.length; looper++) {
+                            Object argument = bootstrapMethodArguments[looper];
+                            if (argument instanceof Type) {
+                                Type type = (Type) argument;
+                                String oldDesc = type.getDescriptor();
+                                String updatedDesc = replaceJavaXwithJakarta(type.getDescriptor());
```

Comment:
We should do javax -> jakarta replacement magic here if, and only if 
type.getSort() returns ARRAY, OBJECT or METHOD int types.

For ARRAY types then we should:
 * check dimensions
 * replace descriptor (if necessary)
 * inspect and potentially replace all elements of this array type // see Type.getElementType()
 * inspect and potentially replace its internal name // see Type.getInternalName()

For METHOD types we should:
 * inspect and potentially replace all arguments of this method type // see Type.getArgumentTypes()
 * inspect and potentially replace its return type // see Type.getReturnType()

For OBJECT types we should:
 * inspect and potentially replace its internal name // see Type.getInternalName()
 * inspect and potentially replace its name // see Type.getClassName()